backend/.env

```
@@ -1,5 +1,5 @@
-DB_PASSWORD=D1ymf8wyQEGthFR1E9xhCq
-DB_USER=LTIdbUser
-DB_NAME=LTIdb
+DB_PASSWORD=password
+DB_USER=postgres
+DB_NAME=ats_db
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_PASSWORD key

(UnorderedKey)